dev-embedded/urjtag/urjtag-2021.03.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
+
+inherit python-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	EGIT_REPO_URI="git://git.code.sf.net/p/urjtag/git"
+	inherit git-r3 autotools
+	S="${WORKDIR}/${P}/${PN}"
+else
+	SRC_URI="mirror://sourceforge/urjtag/${P}.tar.xz"
+	KEYWORDS="amd64 ppc sparc x86"
+fi
+
+DESCRIPTION="Tool for communicating over JTAG with flash chips, CPUs, and many more"
+HOMEPAGE="http://urjtag.sourceforge.net/"
+
+LICENSE="GPL-2"
+SLOT="0"
+
+IUSE="ftdi ftd2xx python readline usb"
```

Comment:
`ftd2xx` doesn't seem to work, at least on my machine.

```
checking for FT_OpenEx in -lftd2xx... no
configure: WARNING: *** libftd2xx not found. No support for FTDI-based USB JTAG cables via libftd2xx.
```

Reply:
I have carefully tested the compilation and running behavior with ``ftd2xx`` enabled and ``ftd2xx`` disabled. When ``ftd2xx`` is enabled, it will automatically depend on ``dev-embedded/libftd2xx``. After installing ``dev-embedded/libftd2xx``, there will be no problem that libftd2xx cannot be found when compiling.

Reply:
If it doesn't find it it doesn't link with it either, and doesn't use it at runtime, so having it installed is useless.

Reply:
After some testing, I found that this is a bug of ``dev-embedded/libftd2xx``. It installed ``libftd2xx.so`` under ``/opt``, and tried to set the ``LDPATH`` global environment variable. This is very inappropriate, even though I did an ``etc-update``. 

The fix was to install it in the correct location. According to the manual https://devmanual.gentoo.org/general-concepts/filesystem/index.html,
the ``/opt`` top-level should only be used for applications that do not conform to the standard filesystem layout. But ``dev-embedded/libftd2xx`` is obviously conform to the standard filesystem layout, so it can be installed to ``usr``, and this bug can be fixed.

I added myself as the acting maintainer of ``dev-embedded/libftd2xx`` and committed a new version.

https://github.com/gentoo/gentoo/pull/32192